src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -403,6 +404,9 @@ public static Glyph CompletionItemKindToGlyph(LSP.CompletionItemKind kind)
             }
         }
 
+        public static ImageElement GlyphToImageElement(Glyph glyph)
```

Comment:
this might fit better as a glyph extension - http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Common/GlyphExtensions.cs,25e63a9ed1eb146f

Since neither of these are LSP types